packages/c_ares.rb

```diff
@@ -3,7 +3,7 @@
 class C_ares < CMake
   description 'C library for asynchronous DNS requests (including name resolves).'
   homepage 'https://c-ares.org/'
-  version '1.23.0'
+  version '1.34.3'
```

Comment:
Missing binaries?

Reply:
Looks like I need to fix the PR generation when building binaries fails... it would help if crew actually exited with an error flag when that happened!